it/it-tests/src/test/java/it/issue/IssueSearchTest.java

```diff
@@ -191,7 +190,7 @@ public void search_issues_by_dates() {
       search(IssueQuery.create().createdBefore(past).createdAfter(today)).list();
       Fail.fail("Expecting 400 from issues search WS");
     } catch (HttpException exception) {
-      assertThat(exception.getMessage()).contains("Start bound cannot be larger than end bound");
+      assertThat(exception.getMessage()).contains("Start bound cannot be larger or equal to end bound");
```

Comment:
Yes, but you could also have added it on existing ones (I've added a smiley because you would have gave me this exact same feedback !)